src/forked/marked/Makefile

```diff
@@ -0,0 +1,12 @@
+all:
```

Comment:
This should probably be deleted as well as we won't be building using `make` but out own build process.